modules/dxp/apps/search-experiences/search-experiences-rest-impl/src/main/java/com/liferay/search/experiences/rest/internal/resource/v1_0/SXPBlueprintResourceImpl.java

```diff
@@ -118,6 +118,8 @@ public Response getSXPBlueprintExport(Long sxpBlueprintId)
 				sxpBlueprint.getTitle(
 					contextAcceptLanguage.getPreferredLocale(), true),
 				".json\"")
+		).header(
+			"Content-Type", "application/json"
```

Comment:
Is `application/xml` is default? So without this the JSON output response would not be able to be parsed. 

If so, don't forget to include the reasoning for certain changes in the commit. You can ask yourself "why does this change need to be committed."